lib/environments/meteorEnvironment.js

```diff
@@ -321,13 +321,7 @@ export default {
     ['.html', '.css'].forEach((ext: string) => {
       const moduleSpecifier = `${basePath}${ext}`;
       if (fs.existsSync(path.join(config.workingDirectory, moduleSpecifier))) {
-        if (config.get('useRelativePaths')) {
-          moduleSpecifiers.push('.'.concat(path.sep, path.basename(moduleSpecifier)));
-        } else {
-          // Strip the leading '.' off of the moduleSpecifier to turn it into a Meteor compliant
-          // absolute path.
-          moduleSpecifiers.push(moduleSpecifier.slice(1));
```

Comment:
> If they change it, they'll be burdened not just with implementing that in their user config, but also with carrying over anything else the environment implemented in moduleNameFormatter.

Perhaps we need to chain these. So instead of overriding the base `moduleNameFormatter` and  the environment `moduleNameFormatter` when you define your own, they all happen. Or, maybe they can make decisions about if the underlying formatter happens (e.g. passing in the next `moduleNameFormatter` functions as an argument, as @trotzig had suggested).

If we do that, it would be pretty easy to keep the option but to implement it in the base `moduleNameFormatter` if we wanted.